embedded/common/modules/sensor-drivers/Driver_Common.h

```diff
@@ -0,0 +1,72 @@
+/* -----------------------------------------------------------------------------
```

Comment:
Why include a copy of Driver_common.h here?  I also see a version in the GPIO pull request in another directory.  I would expect to see these in a common/include/CMSIS folder or something at a similar high level